test/unittests/execute_numeric_test.cpp

```diff
@@ -6,6 +6,7 @@
 #include "instructions.hpp"
 #include "parser.hpp"
 #include <gtest/gtest.h>
+#include <test/unittests/bitcount_test_cases.hpp>
```

Comment:
```suggestion
#include "bitcount_test_cases.hpp"
```

Reply:
Why is `parser.hpp`/etc. included without a path btw?

Reply:
They are not "public" so we just add "internal fizzy include" path to test targets. Which is `lib/fizzy`.